src/main/java/uk/gov/laa/ccms/caab/controller/notifications/ActionsAndNotificationsController.java

```diff
@@ -192,16 +194,16 @@ public String getNotification(
       Model model,
       HttpSession session
   ) {
-    Notification found = notifications.getContent()
-        .stream()
-        .filter(notification -> notification.getNotificationId().equals(notificationId))
+
+    Notification notification = notificationService.getNotification(notificationId,
+        user.getProvider().getId())
+        .blockOptional().stream()
         .findFirst()
         .orElseThrow(() -> new CaabApplicationException(
             String.format("Notification with id %s not found", notificationId)));
```

Comment:
Possible to simplify this now only a single notification is returned

```suggestion
    Notification notification = notificationService.getNotification(notificationId,
        user.getProvider().getId())
        .blockOptional()
        .orElseThrow(() -> new CaabApplicationException(
            String.format("Notification with id %s not found", notificationId)));
```

Reply:
Good spot, have updated